documentation/src/main/asciidoc/topics/proc_configuring_sql_stores_query.adoc

```diff
@@ -1,13 +1,13 @@
 [id='configuring-sql-cache-stores-query_{context}']
-= Configuring SQL query stores
+= Using SQL queries to load data and manipulate persistent storage
 
-Add SQL query cache stores so {brandname} can preload your caches from database tables.
+SQL query cache stores load caches from multiple database tables and use query statements to operate on data.
 
 .Prerequisites
 
-* Provide {brandname} with Protobuf schema that describe any composite keys and values in your database tables.
-+
-{brandname} recommends annotating your Java classes and creating serialization context initializers.
+* Have JDBC connection details. +
+You can add JDBC connection factories directly to your cache configuration. +
+For remote caches in production environments, you should add managed datasources to {brandname} Server configuration and specify the JNDI name in the cache configuration.
 * Remote caches: Copy database drivers to the `server/lib` directory in your {brandname} Server installation.
```

Comment:
Maybe mention the `cli install` tool ? 
`cli install org.postgresql:postgresql:42.1.3`

Reply:
That's a good shout, @tristantarrant I've added a tip in a couple spots.